api/services/wwwConfig.js

```diff
@@ -162,7 +165,11 @@ function _generateEnvs(bag, next) {
   envs = util.format('%s -e %s=%s',
     envs, 'SHIPPABLE_API_TOKEN', bag.serviceUserToken);
   envs = util.format('%s -e %s=%s',
-    envs, 'SHIPPABLE_API_URL', apiUrl);
+    envs, 'SHIPPABLE_API_URL', publicApiUrl);
```

Comment:
will remove this once www is updated to use new ENVs

Reply:
why remove this? `SHIPPABLE_API_URL` is public api url right? Is the reason, to make public api url env more explicit?

Reply:
yes, we want to make it more explicit to the users.